### DISCUSSION TITLE
HomeAssistant trigger on motion?

### DISCUSSION THREAD

**Discussion by ibrewster** [blakeblackshear/frigate#14153] [Ask A Question] [answered]
I've been working on setting up frigate to replace my home-grown solution that is - basically - frigate, just less combined and polished, and as part of that I want to receive "alerts" (by whatever definition of alert I choose to give it - REST Call, MQTT message, etc) in two scenarios:

1) A new object is detected in one of my defined zones. This part was fairly straightforward - I just set up an automation in HomeAssistant, triggered by the "All Count" state for any of my zones changing, with a condition of new value>old value.
2) When an object moves in one of my defined zones.

#2 is the one I am having difficulty with. I'm not seeing anything exposed to HomeAssistant that would indicate motion *of an object* - only motion in general, which doesn't much help.

Is there any way to trigger on object motion? Or set a boolean switch to true if an object is moving in the zones, and false if not (i.e. car parked)? I realize this will result in a lot of notifications if, for example, I'm out mowing the lawn, but that's ideally what I want - basically, if there is something going on, I want to know about it, not just when the something starts.

Alternatively, is there some way to get a notice when a stationary object starts moving, or a moving object becomes stationary? I could probably make that work as well, in combination with the "all count" letting me know when the object in question goes away.

**Comment by NickM-27** [comment, marked as answer]
Use mqtt /reviews topic, this provides what you want

**Reply by ibrewster** [reply]
Ah, indeed it does. I was able to whip up some python code that subscribes to that topic and produces the desired result. Not HomeAssistant, but works for me.

Well, except at night when there is a car parked in my driveway and another car driving by on the road causes the system to completely freak out. But that's a different subject...I'm sure I saw something about that sort of behavior on here before...

Thanks for the pointer!
